ci/build_scripts/package.sh

```diff
@@ -194,6 +194,14 @@ build_virtual_package() {
     if [[ "$PACKAGE_TYPES" =~ apk ]]; then
         env GIT_SEMVER="${APK_VERSION:-$GIT_SEMVER}" RELEASE="r0" nfpm "${COMMON_ARGS[@]}" --packager apk
     fi
+
+    # Expect to build at least 1 virtual package to prevent silent errors
+    # when no virtual packages were created
+    VIRTUAL_PACKAGE_COUNT=$(find target/virtual-packages -type f | wc -l | xargs)
```

Comment:
What's the purpose of the extra `| xargs` with no command using the expended arguments?

Reply:
It just strips whitespace

Reply:
An alternative way of doing this would be to use `tr`, however the command is much longer and always hard to remember the correct whitespace name:

```
echo "   some string    " | tr -d '[:blank:]'
```